tests/gradients/core/test_hadamard_gradient.py

```diff
@@ -1095,7 +1094,7 @@ def cost_fn_param_shift(x):
         assert np.allclose(res_hadamard, res_param_shift)
 
     @pytest.mark.tf
-    @pytest.mark.parametrize("dev_name", ["default.qubit", "default.qubit.tf"])
+    @pytest.mark.parametrize("dev_name", ["default.qubit"])
```

Comment:
Do we ultimately want to test these with other devices such as `lightning.qubit` and the future `reference.qubit`?

Reply:
Probably not.  I was just being lazy when updating these tests.